packages/block-editor/src/components/editor-skeleton/style.scss

```diff
@@ -75,26 +74,6 @@ html.block-editor-editor-skeleton__html-container {
 }
 
 .block-editor-editor-skeleton__sidebar {
```

Comment:
I support the idea of removing the CSS responsible for the different behavior on smaller screens that make it look like an overlay 👍